src/hooks/useLogs.ts

```diff
@@ -365,7 +365,7 @@ export const useLogs = (
         payload: { logsData: queryResponse },
       });
     } catch (error) {
-      if (error.name !== 'AbortError') {
+      if ((error as Error).name !== 'AbortError') {
```

Comment:
Handling the `AbortError` case will probably happen in a few places. I'm not sure what the best pattern is, but what do you think about creating a helper like below?

`const isAbortError = (error): boolean => error instanceof Error && error.name === 'AbortError')`